dataherald/api/fastapi.py

```diff
@@ -408,6 +408,34 @@ def get_response_file(
             file_location,
             media_type="text/csv",
         )
+        
+    @override
+    def get_response_local_file(
```

Comment:
Hi, I like your approach but what do you think about adding this logic in the current `get_response_file` method instead of creating a new one?, it could have the condition to download the file from the local path or s3 as you defined the upload in the `create_sql_query_status.py` file.